index.yaml

```diff
@@ -33,11 +33,11 @@ entries:
     - https://github.com/RoadieHQ/kubewise
     type: application
     urls:
-    - kubewise-0.11.8.tgz
+    - kubewise/kubewise-0.11.8.tgz
```

Comment:
This might break charts installed via automation referencing a specific version but I'm not sure how helm uses the url.